### ISSUE TITLE
Portar Módulo 02

### ISSUE THREAD

**Issue by flaviovdf** [deep-ufmg/praticas-versao-pytorch#2]
- [ ] Criar testes de unidade, não tem testes
- [ ] Código em Pytorch
- [ ] Guiar o aluno
  - [ ] Baselines (random, majority, talvez um scikit-learn simples)
  - [ ] Explicar melhor as bases. Balanceamento de classes, número de features, se precisa normalizar ou não

**Comment by victorbrjorge** [comment]
@flaviovdf Eu tô tentando dar push nas minhas alterações mas tô tomando permission denied. Tem como vc conferir se as permissões estão corretas?

**Comment by flaviovdf** [comment]
Sim, vou fazer aqui
--
Flavio

**Comment by flaviovdf** [comment]
Pronto.
